Readme.md

```diff
@@ -3,8 +3,9 @@
 
 Dockerized for fun and profit.
 
-## Prerequisites
-- Install [Docker](https://docs.docker.com/engine/installation/)
+## Installation instructions for prerequisites
+- [Docker](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04)
```

Comment:
I guess using official docker documentation will be better here i.e. https://docs.docker.com/engine/installation/ 
As it will provide installation instructions for all the platforms